src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/Program.cs

```diff
@@ -5,26 +5,26 @@
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
-using Microsoft.CodeAnalysis;
-using System.Threading.Tasks;
 using Microsoft.DotNet.Cli.Utils;
-using Microsoft.DotNet.ProjectModel;
 using Microsoft.Extensions.CommandLineUtils;
 using Microsoft.Extensions.Internal;
-using Microsoft.Extensions.PlatformAbstractions;
 using NuGet.Frameworks;
+using Microsoft.VisualStudio.Web.CodeGeneration.MsBuild;
+using Microsoft.VisualStudio.Web.CodeGeneration.Utils.Messaging;
 
 namespace Microsoft.VisualStudio.Web.CodeGeneration.Tools
 {
     public class Program
     {
         private static ConsoleLogger _logger;
-        private static bool _isDispatcher;
         private static bool _isNoBuild;
 
         private const string APPNAME = "Code Generation";
         private const string APP_DESC = "Code generation for Asp.net Core";
         private const string TOOL_NAME = "dotnet-aspnet-codegenerator";
+        private const string DESIGN_TOOL_NAME = "dotnet-aspnet-codegenerator-design";
+        private const string PROJECT_JSON_SUPPORT_VERSION = "1.0.0-preview2-update1";
```

Comment:
It's generally not good to have versions hardcoded. Try pulling from the assembly version attribute.